runner/ansible/vars/aws/10-default.yml

```diff
@@ -0,0 +1,27 @@
+---
+
+expected_core:
+  "1.1.1": "30000"
+  "1.1.2": "36000"
+  "1.1.3": "" # unkown value, default may apply
+  "1.1.4": "60"
+  "1.1.5": "6"
+  "1.1.6": "udpu"
+  "1.1.7": "2"
+  "1.1.8": "1"
+  "1.2.1": "true"
+  "1.2.2.sbd": "600"  # unkown value
+  "1.3.1": ""  # doesn't apply for AWS
+  "1.3.2": ""  # doesn't apply for AWS
+  "1.3.4": ""  # doesn't apply for AWS
+  "1.3.5": ""  # doesn't apply for AWS
+  "1.3.6": ""  # doesn't apply for AWS
+  "2.2.1": "SLES_SAP"
+  "2.2.2": "15.1"
+  "2.2.3": "1.1.18"
+  "2.2.3.exclude": "2.0.3+20200511.2b248d828"
+  "2.2.4": "2.4.5" # unkown value, default may apply
```

Comment:
I don't see why these last 3 are unknown. I would expect using the same package versions used in Azure.
By the way, we miss the variable for `2.2.5.exclude`, which would make the check fail